docs/julia.rst

```diff
@@ -0,0 +1,540 @@
+| sgtu256(x:u256, y:u256) -> z:bool           | 1 if x > y, 0 otherwise, for signed numbers in two's complement |
++---------------------------------------------------------------------------------------------------------------+
+| equ256(x:u256, y:u256) -> z:bool            | 1 if x == y, 0 otherwise                                        |
++---------------------------------------------------------------------------------------------------------------+
+| notu256(x:u256) -> z:u256                   | ~x, every bit of x is negated                                   |
++---------------------------------------------------------------------------------------------------------------+
+| andu256(x:u256, y:u256) -> z:u256           | bitwise and of x and y                                          |
++---------------------------------------------------------------------------------------------------------------+
+| oru256(x:u256, y:u256) -> z:u256            | bitwise or of x and y                                           |
++---------------------------------------------------------------------------------------------------------------+
+| xoru256(x:u256, y:u256) -> z:u256           | bitwise xor of x and y                                          |
++---------------------------------------------------------------------------------------------------------------+
+| shlu256(x:u256, y:u256) -> z:u256           | logical left shift of x by y                                    |
++---------------------------------------------------------------------------------------------------------------+
+| shru256(x:u256, y:u256) -> z:u256           | logical right shift of x by y                                   |
++---------------------------------------------------------------------------------------------------------------+
+| saru256(x:u256, y:u256) -> z:u256           | arithmetic right shift of x by y                                |
++---------------------------------------------------------------------------------------------------------------+
+| byte(n:u256, x:u256) -> v:u256              | nth byte of x, where the most significant byte is the 0th byte  |
+| Cannot this be just replaced by and256(shr256(n, x), 0xff) and let it be optimised out by the EVM backend?    |
++---------------------------------------------------------------------------------------------------------------+
+| *Memory and storage*                                                                                          |
++---------------------------------------------------------------------------------------------------------------+
+| mload(p:u256) -> v:u256                     | mem[p..(p+32))                                                  |
++---------------------------------------------------------------------------------------------------------------+
+| mstore(p:u256, v:u256)                      | mem[p..(p+32)) := v                                             |
```

Comment:
Suggestion by @pirapira: We should introduce a type "memory pointer" as a safety feature. This makes it much harder to accidentally swap out the two arguments of `mstore`.